types/diacritics/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for diacritics 1.3
+// Project: https://github.com/andrewrk/node-diacritics
+// Definitions by: Oto Ciulis <https://github.com/otociulis/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function remove(str: string): string;
+export const replacementList: Array<{
+    base: string;
+    chars: string;
+}>;
+
+export const diacriticsMap: {
+    [key: string]: string;
+};
+
+export namespace remove {
```

Comment:
Why do you need remove namespace ? Dont see any mention in https://github.com/andrewrk/node-diacritics/blob/master/index.js